modules/bitgo/test/v2/unit/tss/eddsa.ts

```diff
@@ -397,7 +398,7 @@ describe('test tss helper functions', function () {
 
     describe('offerUserToBitgoRShare:', async function() {
       it('should succeed to send Signature Share', async function() {
-        const signatureShare = { from: 'user', to: 'bitgo', share: validUserSignShare.rShares[3].r + validUserSignShare.rShares[3].R } as SignatureShareRecord;
+        const signatureShare = { from: 'user', to: 'bitgo', share: validUserSignShare.rShares[3].v + validUserSignShare.rShares[3].r + validUserSignShare.rShares[3].R } as SignatureShareRecord;
```

Comment:
Will this change be backward compatible for existing uses who do not have "v" in keys

Reply:
Yes, the v field in the signing share is generated from their signing key at the time of signing.